From 119d9cb8b44b39dafd8bf9719c4370ed64383f06 Mon Sep 17 00:00:00 2001
From: Jeevaka Prabu Badrappan <jeevaka.badrappan@intel.com>
Date: Sat, 14 Mar 2020 13:39:34 +0530
Subject: [PATCH] kernel: Disable ASM_GOTO support

As clang toolchain lacks ASM_GOTO support, remove the check
to build kernel successfully

Tracked-On: OAM-90323
Signed-off-by: Jeevaka Prabu Badrappan <jeevaka.badrappan@intel.com>
---
 arch/x86/Makefile | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/x86/Makefile b/arch/x86/Makefile
index 4833dd7e2cc0..f36dbe7872ad 100644
--- a/arch/x86/Makefile
+++ b/arch/x86/Makefile
@@ -306,10 +306,10 @@ vdso_install:
 
 archprepare: checkbin
 checkbin:
-ifndef CONFIG_CC_HAS_ASM_GOTO
-	@echo Compiler lacks asm-goto support.
-	@exit 1
-endif
+#ifndef CONFIG_CC_HAS_ASM_GOTO
+#	@echo Compiler lacks asm-goto support.
+#	@exit 1
+#endif
 ifdef CONFIG_RETPOLINE
 ifeq ($(RETPOLINE_CFLAGS),)
 	@echo "You are building kernel with non-retpoline compiler." >&2
-- 
2.17.1

